src/commands/upgrade/themeupgrader.js

```
@@ -18,9 +18,9 @@ const git = simpleGit();
  * then handles the upgrade accordingly.
  */
 class ThemeUpgrader {
-  constructor(jamboConfig) {
+  constructor(jamboConfig = {}) {
     this.jamboConfig = jamboConfig;
-    this._themesDir = jamboConfig.dirs.themes;
+    this._themesDir = jamboConfig.dirs && jamboConfig.dirs.themes;
```

should we throw an error if there is no themes directory in the jambo config?

---

We're currently not doing that for any other command that uses the themes directory. My understanding is that once jambo init has been run we're guaranteed to have a valid jambo config from `jambo.json` (please correct me if I'm wrong), so we're safe here.

---

ok sgtm!

---

Create this [item](https://yexttest.atlassian.net/browse/SLAP-816) to track the work!